HTTP Error 404: Not Found

### Before submitting a bug report...

- [x] This bug wasn't already reported.
(I have checked every bug report on GitHub)


### Title

- [X] The title is no longer "[BUG] Title" and I edited it with the right error name.


### Describe the bug

I cant seem to use undetected_chromedriver and I cant find out why since this only started happening today.

### Copy and paste your error

2024-02-15 08:01:22,241 [ERROR] HTTPError: HTTP Error 404: Not Found
Traceback (most recent call last):
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\main.py", line 47, in main
    earned_points = executeBot(currentAccount, notifier, args)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\main.py", line 221, in executeBot
    with Browser(mobile=False, account=currentAccount, args=args) as desktopBrowser:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\src\browser.py", line 41, in __init__
    self.webdriver = self.browserSetup()
                     ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\farmer2\MS-Rewards-Farmer\src\browser.py", line 86, in browserSetup
    driver = webdriver.Chrome(
             ^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\seleniumwire\undetected_chromedriver\webdriver.py", line 61, in __init__
    super().__init__(*args, **kwargs)
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\__init__.py", line 258, in __init__
    self.patcher.auto()
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\patcher.py", line 178, in auto
    self.unzip_package(self.fetch_package())
                       ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\site-packages\undetected_chromedriver\patcher.py", line 287, in fetch_package
    return urlretrieve(download_url)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 240, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
                            ^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 215, in urlopen
    return opener.open(url, data, timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 521, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 630, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 559, in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 492, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "C:\Users\james\AppData\Local\Programs\Python\Python312\Lib\urllib\request.py", line 639, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

### Screenshots

the window will not open.

### Value of dashboard variable

https://gist.github.com/Xtrau505/b2b21785ab2b6deb5f3c2a803b4e6250

--- a/Dockerfile
+++ b/Dockerfile
@@ -24,6 +24,8 @@ RUN pip install --root-user-action=ignore -r requirements.txt
 ENV DISPLAY=:99
 ENV DBUS_SESSION_BUS_ADDRESS=/dev/null
 
+RUN sed -i "s|download_url = \"https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/%s/%s/%s\"|download_url = \"https://storage.googleapis.com/chrome-for-testing-public/%s/%s/%s\"|g" /usr/local/lib/python3.10/dist-packages/undetected_chromedriver/patcher.py
+
 # copy entrypoint script
 COPY entrypoint.sh ./
 

